### DISCUSSION TITLE
Don't understand why my video won't play?

### DISCUSSION THREAD

**Discussion by DaniGenerous** [community/community#87008] [New to GitHub] [answered]
Hello! I am confused as to why my video on my published page does not play... it is just a black screen (No alt message shows) It is towards the end of this page of my site:  https://danigenerous.github.io/CombatingErasure/individualindex.html

**Comment by ghost** [comment]
Hi @DaniGenerous I checked the website but the video works fine and plays well may be its a problem on your end or a small network issue

![Screenshot 2024-01-14 003307](https://github.com/community/community/assets/121353576/a04ef711-e329-49bd-9a2d-55b549073645)

**Reply by ghost** [reply]
As you can see I played the video...

**Reply by ghost** [reply]
Dude sry for that but now only i saw there is another video with the issue you mentioned. I guess it might be a problem with the video try another video extension.

**Reply by ghost** [reply]
I'm saying this as the alt text did not load up but yet the video does not play... That is why

**Reply by DaniGenerous** [reply]
Haha I was just typing that when you responded. Thats my bad, i should've been more specific

**Comment by seyLu** [comment]
#### Hi @DaniGenerous,

Whenever something is wrong with your site, just right-click + inspect or via the keyboard shortcut <kbd><kbd>ctrl</kbd><kbd>shift</kbd><kbd>i</kbd></kbd> to open the devtools.

![image](https://github.com/community/community/assets/98249191/d7e010ae-37aa-42c6-a6b4-0b27803060e2)

And if we look at your repo, try to search for any `.mp4` files:

![image](https://github.com/community/community/assets/98249191/2d471669-ba56-4cf5-bdea-5a17d6a4a4b4)

or any files named `delbertfilm`:

![image](https://github.com/community/community/assets/98249191/703d05b0-3436-4b09-83e5-df2203a005bb)


Make sure the file is uploaded to the repo first. :)

**Reply by ghost** [reply]
Then why the alt text is not showing instead just a black screen

**Reply by seyLu** [reply, marked as answer]
[Videos in HTML 5, unlike images, don't have alt text](https://stackoverflow.com/questions/32129240/html5-video-element-alternative-text).

**Reply by ghost** [reply]
Thank You @seyLu till now i thought the text btwn the video tags is actually the  alt text but it is actually the title thank you s much learned a new thing...

**Reply by DaniGenerous** [reply]
As did I! Thank you @seyLu for looking into this for me!
